pkg/logs/launchers/integration/launcher.go

```diff
@@ -157,56 +253,117 @@ func (s *Launcher) makeFileSource(source *sources.LogSource, logFilePath string)
 
 // TODO Change file naming to reflect ID once logs from go interfaces gets merged.
 // createFile creates a file for the logsource
-func (s *Launcher) createFile(id string) (string, error) {
-	logFilePath := s.integrationLogFilePath(id)
+func (s *Launcher) createFile(source string) (*FileInfo, error) {
+	filepath := s.integrationLogFilePath(source)
 
-	file, err := os.Create(logFilePath)
+	file, err := os.Create(filepath)
 	if err != nil {
-		return "", nil
+		return nil, nil
```

Comment:
Should this be returning the error? `return nil, error`? 
If this function fails, we return `nil`. I think his will panic on line `128` when you access `logFile.filename`